frontend/src/index.scss

```diff
@@ -8,3 +8,6 @@ $lightTxtColor: #efefef;
 //     font-family: Arial, Helvetica, sans-serif;
 // }
 
+.ant-form-item {
+    margin-bottom: 0px;
```

Comment:
🤔 The default sucks?

Reply:
Too much spacing, it's defaulted to 24px